hxinsight-extension/src/test/java/org/alfresco/hxi_connector/hxi_extension/service/HxInsightClientTest.java

```diff
@@ -255,4 +258,51 @@ void shouldSet503StatusCodeOnCheckedException_Agents()
         WebScriptException exception = assertThrows(WebScriptException.class, hxInsightClient::getAgents);
         assertEquals(SC_SERVICE_UNAVAILABLE, exception.getStatus());
     }
+
+    @Test
+    @SneakyThrows
+    void canSubmitFeedbackWithoutException()
+    {
+        // given
+        HttpResponse response = mock(HttpResponse.class);
+        given(response.statusCode()).willReturn(200);
+
+        given(httpClient.send(any(), any())).willReturn(response);
+
+        // when
+        hxInsightClient.submitFeedback("dummy-id-1234", new Feedback(GOOD, "This answer was amazing"));
```

Comment:
You can also verify that using:
`Throwable throwable = catchThrowable(() -> hxInsightClient.submitFeedback("dummy-id-1234", new Feedback(GOOD, "This answer was amazing"));`
and then
`assertThat(throwable).doesNotThrowAnyException();`
This will close the PMD's mouth and allow to avoid warning suppression

Reply:
I think it's neater to just suppress the warning in this instance.

Reply:
👌 I'm fine with that